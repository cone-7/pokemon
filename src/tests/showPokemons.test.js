import React from 'react';
import ShowPokemons from '../components/showPokemons.js';
import renderer from 'react-test-renderer';

test('Should render pokemons', () => {

  const pokemons = [
      {name:'ditto', img: "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/132.png"}
  ]
  const component = renderer.create(
    <ShowPokemons pokemons={pokemons}></ShowPokemons>,
  );
  
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});