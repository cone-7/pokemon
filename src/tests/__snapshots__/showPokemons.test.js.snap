// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render pokemons 1`] = `
<div>
  <div
    className="pokemon-card"
  >
    <span>
      ditto
    </span>
    <img
      alt=""
      src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/132.png"
    />
  </div>
</div>
`;
